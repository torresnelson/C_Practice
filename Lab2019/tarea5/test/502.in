# prueba de hash: actualizar_hash, eliminar_de_hash  
asociar_en_hash 5 vvv
asociar_en_hash 85 axss
valor_en_hash 5
valor_en_hash 85
actualizar_hash 5 ppp
actualizar_hash 85 axss
valor_en_hash 5
valor_en_hash 85
asociar_en_hash 25 sss
valor_en_hash 25
actualizar_hash 25 vvv
valor_en_hash 25
valor_en_hash 25
asociar_en_hash 3 ppp
asociar_en_hash -3 ppp
asociar_en_hash 465 frtt
asociar_en_hash -2753 bgtt
asociar_en_hash 415 xsde
asociar_en_hash 63 juiuk
asociar_en_hash 185 xss
esta_lleno_hash
actualizar_hash 25 ooo
existe_asociacion 25
valor_en_hash 25
eliminar_de_hash 25
esta_lleno_hash
existe_asociacion 25
asociar_en_hash 25 www
valor_en_hash 25
valor_en_hash 5
valor_en_hash 85
eliminar_de_hash 25
eliminar_de_hash 85
eliminar_de_hash 5
existe_asociacion 25
existe_asociacion 85
existe_asociacion 5
valor_en_hash 3
valor_en_hash -3
eliminar_de_hash 3
eliminar_de_hash -3
asociar_en_hash 3 ddd
asociar_en_hash -3 ddd
valor_en_hash 3
valor_en_hash -3
Fin


