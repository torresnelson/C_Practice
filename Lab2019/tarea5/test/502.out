1>#  prueba de hash: actualizar_hash, eliminar_de_hash  .
2>Establecida la asociación.
3>Establecida la asociación.
4>vvv
5>axss
6>Actualizada la asociación.
7>Actualizada la asociación.
8>ppp
9>axss
10>Establecida la asociación.
11>sss
12>Actualizada la asociación.
13>vvv
14>vvv
15>Establecida la asociación.
16>Establecida la asociación.
17>Establecida la asociación.
18>Establecida la asociación.
19>Establecida la asociación.
20>Establecida la asociación.
21>Establecida la asociación.
22>LLeno.
23>Actualizada la asociación.
24>Existe asociacion.
25>ooo
26>Eliminada la asociación.
27>No lleno.
28>No existe asociacion.
29>Establecida la asociación.
30>www
31>ppp
32>axss
33>Eliminada la asociación.
34>Eliminada la asociación.
35>Eliminada la asociación.
36>No existe asociacion.
37>No existe asociacion.
38>No existe asociacion.
39>ppp
40>ppp
41>Eliminada la asociación.
42>Eliminada la asociación.
43>Establecida la asociación.
44>Establecida la asociación.
45>ddd
46>ddd
47>Fin.
